modules/common/src/main/java/org/eclipse/xpanse/common/aop/HttpRequestInfoManager.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.common.aop;
+
+
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.Objects;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.models.common.enums.Csp;
+import org.springframework.web.context.request.RequestAttributes;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+/**
+ * Bean to provide methods to manage request attributes.
+ */
+@Slf4j
+public class HttpRequestInfoManager {
```

Comment:
Is this class really required? We do not seem to use the CSP from http request attribute.